tox.ini

```diff
@@ -79,17 +76,14 @@ commands =
 
 [testenv:travis]
 basepython = python3.6
-passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY GANACHE_BINARY
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY GANACHE_BINARY BRANCH
```

Comment:
Which library needs the BRANCH env variable?

Reply:
No library, have a look at the installation of d3a-interface on the setup.py file though. The reason for that is for the dockerfile to use an environment variable to select the correct d3a-interface branch (master, production or live). The change in Travis is needed in order to test against the correct d3a-interface branch according to merge target.

Reply:
Ahhh. ok! Now with your d3a-web PR it makes sense. Thanks!